src/restic/backend/sftp/config.go

```diff
@@ -11,58 +12,47 @@ type Config struct {
 	User, Host, Dir string
 }
 
-// ParseConfig extracts all information for the sftp connection from the string s.
+// ParseConfig parses the string s and extracts the sftp config. The two
+// supported configuration formats are sftp:user@host:port/directory and
+// sftp:user@host:directory. The directory will be path Cleaned and can be
+// an absolute path.
```

Comment:
I think the comment is not accurate: What about the format `sftp://user@host`? The prefix isn't mentioned at all...